.travis.yml

```diff
@@ -139,6 +140,7 @@ install:
     yarn global add grunt-cli
     yarn install
   fi
+- if [[ "$SECURITY" == "1" ]]; then wget -P $SECURITYCHECK_DIR https://get.sensiolabs.org/security-checker.phar && chmod +x $SECURITYCHECK_DIR/security-checker.phar;fi
```

Comment:
There is no space in between `;` and `fi` (on the unchanged lines in this file, there is a space).

Reply:
Sorry, but that's irrelevant as it works either way.